spec/models/aws/s3_storage_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Aws::S3Storage do
+  # allow(:storage_double) {double}
+  # allow(Aws::S3Strorage).to receive(:new).and_respond_with(storage_double)
```

Comment:
remove these comments

Reply:
have removed comments and pushed changes